### PR TITLE
Update to list consistency and clarifying wording 

### PR DESCRIPTION
**Pull request by Ybittan**
The majority of this PR involves creating consistency within the lists, such as capitalization and punctuation.

### CODE REVIEW

File: core/baseline-core-v1.0-psd01.md [hunk 1]
```diff
@@ -2097,9 +2097,9 @@ The quorum required to accept or reject a joint state update on an IVSM MUST be
 The joint state object on an IVSM MUST be updated on said IVSM based on each received BPI Subject vote, either accept or reject.
 
 #### **[R200]** 
-Once the updated joint state is either accepted or rejected based on the defined quorum, an IVSM MUST 
-* notify all BPI Subjects if the joint state has been finalized based on the rules of the State Synchronization and Advancement Predicate of the IVSM
-* cryptographically seal the joint state such that no further updates to the joint state can be processed.
+Once the updated joint state is either accepted or rejected based on the defined quorum, an IVSM MUST: 
+* Notify all BPI Subjects if the joint state has been finalized based on the rules of the State Synchronization and Advancement Predicate of the IVSM
+* Nryptographically seal the joint state such that no further updates to the joint state can be processed
```
Comment (Therecanbeonlyone1969) [inline core/baseline-core-v1.0-psd01.md:2102]: @Ybittan the N is a tad much!

**Review by Therecanbeonlyone1969 — CHANGES_REQUESTED** [review]
see my comment

File: core/baseline-core-v1.0-psd01.md (hunk 1 above)
Comment (Ybittan) [reply core/baseline-core-v1.0-psd01.md:2102]: You caught my red herring

**Review by Therecanbeonlyone1969 — APPROVED** [review]
lgtm

**Review by kthomas — APPROVED** [review]

**Pull request merged**